app/views/student_teams/_team_display.html.erb

```diff
@@ -0,0 +1,71 @@
+<table style="width:80%" align="center" >
+    <tr style="border: 1px outset #000000; padding: 10px 20px" >
+        <th class="head">Username</th>
+        <th class="head">Full name</th>
+        <% if @student.assignment.duty_based_assignment? %>
+            <th class="head">Role</th>
+        <% end %>
+        <th class="head">Email address</th>
+
+        <% if @teammate_review_allowed %>
+            <th class="head"><%=t ".review_action"%></th>
+        <% end %>
+    </tr>
+
+    <% @student.team.participants.each do |member| %> <!--start team member content-->
+        <tr>
+            <td><%= member.user.name(session[:ip]) %></td>
```

Comment:
Information of the user should be fetched through participant_id.

Reply:
Updated 👍